### PR TITLE
Pass already existing UIDs set to fixParseSuccessUIDs

### PR DESCRIPTION
**Pull request by Rheeseyb**
**Problem:**
The function used to map old UIDs to new UIDs on parse results was not taking into account the full set of existing UIDs, meaning there was a case where wrapping an element in another could result in duplicate UIDs (as we would try to add the old UID to the new element, but also keep that same old UID on the original element).

**Fix:**
We already have the set of all existing UIDs as part of the parse, so we can simply provide that to the `fixParseSuccessUIDs` function and use that to prevent these kinds of incorrect mappings. I've also added a test case to capture this situation.

**Bonus Fix:**
We previously discussed that using `--maxWorkers '50%'` makes the tests more stable, so since I was bitten by an arbitrarily failing test whilst working on this I've decided to add that to the `test` script.

### CODE REVIEW

File: editor/src/core/workers/parser-printer/uid-fix.spec.ts [hunk 1]
```diff
@@ -183,6 +183,27 @@ describe('fixParseSuccessUIDs', () => {
     )
     expect(getUidTree(firstResult)).toEqual(getUidTree(secondResult))
   })
+
+  it('handles newly inserted duplicate elements without duplicating uids', () => {
```
[the remark this answers is not in the document]
Comment (Rheeseyb) [reply editor/src/core/workers/parser-printer/uid-fix.spec.ts:187]: Yes :)

[the remark this answers is not in the document]
Comment (Rheeseyb) [reply editor/src/core/workers/parser-printer/uid-fix.spec.ts:187]: Without the fix the output is:
```
    "4ed
        random-uuid
      001
        c85
          c85
            random-uuid
      storyboard
        scene
          component"
```
So the parser will give the new element the UID `593`, but will then "fix" it with the UID `c85`.

**Comment by seanparsons** [comment]
> Bonus Fix:
Is this related to this PR?

**Review by seanparsons — APPROVED** [review]

**Comment by Rheeseyb** [comment]
> > Bonus Fix:
> 
> Is this related to this PR?

Nope. I can rip that out and commit straight to master separately as it has no effect on the project itself, it is literally just the change to the `test` script in the editor's `package.json`

**Review by gbalint — APPROVED** [review]

**Pull request merged**